### PR TITLE
refactor: move setup and assertions in 'testsupport'

### PR DESCRIPTION
**Pull request by xcoulon**
Moved the setup and assertions funcs in `testsupport` so we can later introduce a `test/perf` package and reuse the funcs.

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: testsupport/user_assertions.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+func VerifyMemberStatus(t *testing.T, memberAwait *wait.MemberAwaitility) {
+	_, err := memberAwait.WaitForMemberStatus(wait.UntilMemberStatusHasConditions(ToolchainStatusReady()))
+	require.NoError(t, err, "failed while waiting for MemberStatus")
+}
+
+func VerifyToolchainStatus(t *testing.T, hostAwait *wait.HostAwaitility) {
+	_, err := hostAwait.WaitForToolchainStatus(wait.UntilToolchainStatusHasConditions(ToolchainStatusReady()))
+	require.NoError(t, err, "failed while waiting for ToolchainStatus")
+}
```
Comment (MatousJobanek) [inline testsupport/user_assertions.go:113]: You have put it into the the `user_assertion.go` file, but this is not related to user management verifications

Comment (xcoulon) [reply testsupport/user_assertions.go:113]: ah right, let me move them in a `toolchain_status.go` file (or something similar)

**Comment by xcoulon** [comment]
> Awesome idea 👍
> btw, you could use the "dot import" to make the lines of the code a bit shorter, but it's a detail :-)

good idea, let me check that

**Comment by xcoulon** [comment]
> > Awesome idea 👍
> > btw, you could use the "dot import" to make the lines of the code a bit shorter, but it's a detail :-)
> 
> good idea, let me check that

done in 4d1424c

File: testsupport/user_assertions.go (hunk 1 above)
Comment (xcoulon) [reply testsupport/user_assertions.go]: done in 4d1424c: funcs moved into `testsupport/toolchain_status_assertions.go`
